packages/audio-block/tailwind.config.js

```diff
@@ -0,0 +1,9 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+module.exports = {
+    presets: [require('@frontify/fondue/tailwind')],
```

Comment:
```suggestion
    presets: [require('@frontify/fondue/tailwind'), require('../../tailwind.config')],
```

Need to include tailwind tokens from the root so hover and some other styles are working as intended.

Reply:
Added within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/270d443a55f7aca51132ff498bb0c1cdedbc3b59)